src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  tooltip: {
+    padding: '6px',
+    background: 'rgba(0, 0, 0, .8)',
+    color: 'white',
+    borderRadius: '4px',
+    pointerEvents: 'none',
+  },
+  tooltipKey: {
+    display: 'inline-block',
+    width: '10px',
+    height: '10px',
+    marginRight: '10px',
+  },
+};
+
+function CustomTooltip({ classes, tooltipModel, series }) {
+  if (tooltipModel.opacity === 0) return null;
+
+  const currPoint = tooltipModel.dataPoints[0];
+  const labelColors = tooltipModel.labelColors[0];
+  const { index } = currPoint;
+  const currSeries = series[currPoint.datasetIndex];
+  const higherIsBetter = !currSeries.meta.lower_is_better;
+  const higherOrLower = higherIsBetter ? 'higher is better' : 'lower is better';
+  const curr = currSeries.data[index];
+  const hgURL = `https://hg.mozilla.org/mozilla-central/pushloghtml?changeset=${
+    curr.revision
+  }`;
```

Comment:
use this

```javascript
  const hgURL = `https://hg.mozilla.org/mozilla-central/pushloghtml?${Object2URL({changeset: curr.revision})}`;
```

Reply:
I'm getting `'Object2URL' is not defined`, should I define that function with the purpose that you are trying to get?